File: Interviews/LeetCode/Daily-Problems/2022/November/Day-11/26.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i, j = 1, 1
        while j < len(nums):
            if nums[j] != nums[j - 1]:
                nums[i] = nums[j]
                i += 1
            j += 1

        return i

File: Interviews/LeetCode/Daily-Problems/2022/November/Day-11/26.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 1, j = 1;
        while (j < nums.size()) {
            if (nums[j] != nums[j - 1]) {
                nums[i] = nums[j];
                i += 1;
            }
            j += 1;
        }
        
        return i;
    }
};